packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -428,7 +391,9 @@ class ScrollableBase extends Component {
 				rDirection = reverseDirections[direction],
 				viewportBounds = containerRef.getBoundingClientRect(),
 				focusedItemBounds = focusedItem.getBoundingClientRect(),
-				endPoint = this.getEndPoint(direction, focusedItemBounds, viewportBounds);
+				endPoint = (direction === 'up') ?
+					{x: focusedItemBounds.left + focusedItemBounds.width / 2, y: viewportBounds.top + focusedItemBounds.height / 2 - 1} :
+					{x: focusedItemBounds.left + focusedItemBounds.width / 2, y: viewportBounds.top + viewportBounds.height - focusedItemBounds.height / 2 + 1};
```

Comment:
`x` is always the same expression, so let's put comparison into `y` value's expression.